webapp/src/components/viewHeader/filterValue.tsx

```diff
@@ -31,7 +31,7 @@ const filterValue = (props: Props): JSX.Element|null => {
         displayValue = filter.values.map((id) => {
             const option = template.options.find((o) => o.id === id)
             return option?.value || '(Unknown)'
-        }).join(', ')
+        }).join(' or ')
```

Comment:
This is going to generate problems with the translations (probably the previous one would generate the same kind of problems. We should file a ticket (probably a help wanted one) to see how we tackle this on translations.

Reply:
So notion doesn't handle it all (only allowed to select one option at one time) 
![image](https://user-images.githubusercontent.com/17804942/121903966-cc819e80-ccf6-11eb-805f-be4f0dd2bfd1.png)

Jira handles it how we handle it currently
![image](https://user-images.githubusercontent.com/17804942/121904144-f044e480-ccf6-11eb-8b8f-d17ef437a586.png)

Asana doesn't handle it either (similar to notion). It's a open feature request.
https://forum.asana.com/t/sort-and-filter-project-by-multiple-fields-due-date-and-assignee-for-example/15234/63

![image](https://user-images.githubusercontent.com/17804942/121904553-49ad1380-ccf7-11eb-8b7a-3ccd5ea2708c.png)


So with all that in mind, I've gone ahead and reverted this change to match Jira's.

Reply:
:+1: